packages/ui-components/.storybook/tailwind.css

```diff
@@ -0,0 +1,7 @@
+@import 'https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;1,100;1,200;1,300;1,400;1,500;1,600;1,700&family=Open+Sans:ital,wght@0,300..800;1,300..800&display=swap';
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
+@import '../../../apps/site/styles/base.css';
```

Comment:
It feels weired that the ui components package depends on another package's styles (the website) -- that should definitely not be the case. These styles should live here if nothing else.

Reply:
(But I see this is only for storybook, so 🤷)